oggm/utils/_workflow.py

```diff
@@ -2321,7 +2321,13 @@ def np_convert(o):
 
     def get_climate_info(self, input_filesuffix=''):
         """Convenience function handling some backwards compat aspects"""
-
+            
+        # I don't know another way right now, 
+        # as I was not able to get the input_filesuffix 
+        # to those functions that use get_climate_info 
+        if cfg.PARAMS['baseline_climate'] == 'ERA5_daily':
+             input_filesuffix = '_daily'
```

Comment:
that's really not clean indeed. I'm not sure to understand yet where you need that, but maybe it's something to delegate to the mass-balance models which are calling that?

Reply:
Or, if that's too many changes in code, at least you can do:

```python
        if not input_filesuffix and (cfg.PARAMS['baseline_climate'] == 'ERA5_daily'):
             input_filesuffix = '_daily'
```

Reply:
I will use your proposition. The problem is that I don't understand how `get_climate_info()` is used in OGGM when an input_filesuffix is used (e.g. in get_ref_mb_data, climate.py, massbalance.py). How does it make sure that the right climate is used? If I have now also a daily climate file and want to use the same methods, (get_ref_mb_data...), it had happened that the wrong climate file is used for get_climate_info(), therefore I needed to change this in OGGM, but I guess I misunderstand here something.